document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -1167,7 +1172,7 @@ private String getColumnValue(
       }
 
       // check for ID column
-      if (Set.of(ID, IMPLICIT_ID_ALIAS).contains(columnName)) {
+      if (OUTER_COLUMNS.contains(columnName) || IMPLICIT_ID_ALIAS.equals(columnName)) {
```

Comment:
What was the issue? Wasn't the existing condition sufficient - `Set.of(ID, IMPLICIT_ID_ALIAS).contains(columnName)`?

Reply:
Discussed offline, it was needed with an empty selection.